### PR TITLE
Ansible script to test using hwcct

### PR DESCRIPTION
**Pull request by persiaAziz**
No need to merge now

### CODE REVIEW

File: deploy/vm/test/hwcct/hwcct_playbook.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            get_url: {{ sapcar_url }}
+            dest: /hana/shared/SAPCAR.EXE
+          - name: change permission of SAPCAR.EXE
+            file: /hana/shared/SAPCAR.EXE mode=a+x
+          - name: download HWCCT
+            get_url: {{ hwcct_url }}
+            dest: /hana/shared/hwcct.SAR
+          - name: extract hwcct.SAR using SAPCAR
+            shell: ./SAPCAR.EXE -xf HWCCT_122_17-20011536.SAR hwcct
+            args:
```
Comment (pabowers) [inline deploy/vm/test/hwcct/hwcct_playbook.yml]: add a creates argument so that the call will be idempotent and know if it had been previously run

Comment (persiaAziz) [reply deploy/vm/test/hwcct/hwcct_playbook.yml]: good catch!

File: deploy/vm/test/hwcct/hwcct_config.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+"use_hdb":false,
+"blades":["localhost"],
```
Comment (tniek) [inline deploy/vm/test/hwcct/hwcct_config.json]: nit: spacing between key and value (i.e. write "use_hdb":SPACEfalse instead of "user_hdb":false)

Comment (tniek) [reply deploy/vm/test/hwcct/hwcct_config.json]: nit: also, need to review naming convention (camel case?)

**Pull request merged**